app/services/app/app.ts

```diff
@@ -152,7 +153,6 @@ export class AppService extends StatefulService<IAppState> {
 
     this.crashReporterService.endStartup();
 
-    this.FINISH_LOADING();
```

Comment:
While debugging this issue, I discovered that we were removing the loading screen too early due to not consistently using the decorator.